aws-lambda-python/lambda_function.py

```diff
@@ -0,0 +1,4 @@
+from rook import auto_start
```

Comment:
I'm pretty sure that's not the way to use Rookout under Lambda :(

Reply:
https://github.com/Rookout/Rookout/pull/580/files
Just saw this PR from ori, guess there was something to add to make it work. I'll wait until this is merged and I test it with this example